src/Desktop.Analyzers/Core/DesktopAnalyzersResources.resx

```diff
@@ -368,7 +368,7 @@
     <comment>CA3075</comment>
   </data>
   <data name="DoNotUseDtdProcessingOverloadsMessage" xml:space="preserve">
-    <value>{0} uses unsafe overload of '{1}' method</value>
+    <value>Uses unsafe overload of '{0}' method</value>
```

Comment:
Why this change? Starting with uses sounds pretty blunt and without the context, there would be no way of distinguishing multiple instances of this error in the error list.